docker/docker-compose.dev.yml

```diff
@@ -18,6 +18,8 @@ services:
       dockerfile: Dockerfile
       args:
         APP_ENV: dev
+    depends_on:
+      - elasticsearch:
```

Comment:
will this work with `:` at the end?

Reply:
Nice catch! 

In general, idk about these root-level compose files. Are they still being used? They certainly haven't gotten some updates in awhile.